### ISSUE TITLE
Map keyboard shortcut to different RDP saved profiles?

### ISSUE THREAD

**Issue by davkav** [FreeRDP/FreeRDP#3467]
I am using a HP thin client that is running thin pro OS with FreeRDP.

The idea is that we have a thin client configured with 4 RDP sessions that are always active. We need a way for the operator to easily switch between profiles. 

Is there a way to map a keyboard shortcut, Ctrl+F1 for example, to the first RDP profile configured. 
Then map Ctrl+F2 to the second and so on.

I am aware that Ctrl+Alt+Tab is configured out of the box on the thin client to cycle between active windows on thin pro but the popup that displays the list of available windows to cycle through displays the IP address of the RDP Sessions and not the profile name that is configured in Thin Pro.

All of my RDP Sessions use the same terminal server but with different usernames.

Can anyone help me out? 

Thanks in advance,
Dave

**Comment by FragsterAt** [comment]
in linux it may be done with several x-sessions with ctrl+alt+f7, ..f8, etc switch.

**Comment by davkav** [comment]
Thanks Fragster but because of the RDP profile configuration in HP thin pro, the switch between each profile needs to be done in the one session

**Comment by FragsterAt** [comment]
I think, freerdp need customizable keyboard shortcuts, which are processed on the host system,
eg i have shortcut to lock host system and want, and I want it to work in host system when I'm in freerdp session. Other shortcuts must be processed in session.

**Comment by bmiklautz** [comment]
@davkav as I read it this is more something related to thin pro os than to FreeRDP?

**Comment by akallabeth** [comment]
Closing due to lack of feedback

**Comment by davkav** [comment]
Sorry lads, I should have come back and updated this.

I ended up starting and leaving open 4 RDP profiles on the thin client. I then mapped the keyboard shortcut ctrl+tab to cycle between active windows on the thin client ( I didnt want to use Alt+tab as that would interfere with the windows session within the RDP connection).

I labelled each of the RDP IP addresses in the hosts file with 'friendly' names and now when the user cycles through the windows on the thin client a popup appears listing the active connections.
